pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1014,15 +1032,16 @@ func appNetworkDoActivateUnderlayNetwork(
 		}
 	}
 
-	bridgeIPAddr, appIPAddr, err := getUlAddrs(ctx, ulNum-1,
-		status.AppNum, ulStatus, netInstStatus)
+	appID := status.UUIDandVersion.UUID
+	appIPAddr, err := getUlAddrs(ctx, ulNum-1, netInstStatus, ulStatus,
```

Comment:
You removed the publish from getUlAddres, and I don't know whether there is some code below this line which depends on being able to retrieve what was published. So I think it makes sense to be conservative and restore the publish calls where they were (with the addition being for the separate allocation of the bridgeIPAddr and publishing after that).

Reply:
added back.